src/components/PlanningTool.jsx

```diff
@@ -805,6 +805,8 @@ PlanningTool.propTypes = {
       canResize: PropTypes.oneOf(['both', 'left', 'right', false]),
       minimumDuration: PropTypes.oneOfType([PropTypes.number, PropTypes.oneOf([false])]),
       maximumDuration: PropTypes.oneOfType([PropTypes.number, PropTypes.oneOf([false])]),
+      workTime: PropTypes.number,
```

Comment:
@LaChope Not sure how this behaves, just remember that we are building a generic component which should work even if last level of resource-s does not represent actual work-sessions ... If this requirement is achieved here then ok.